python/ray/serve/handle.py

```diff
@@ -145,7 +145,7 @@ def _set_request_protocol(self, request_protocol: RequestProtocol):
             _request_protocol=request_protocol
         )
 
-    def _get_or_create_router(self) -> Union[Router, asyncio.AbstractEventLoop]:
+    def _get_or_create_router(self) -> Tuple[Router, asyncio.AbstractEventLoop]:
```

Comment:
See below, this returns a tuple of these two types, not one-or-the-other

Reply:
yeah, probably just a typo :)

really wish we were actually using mypy on the serve codebase...

Reply:
I would love that!

It's definitely tricky around the actual `.remote()` calls and whatnot that users would see, but it would be great just for internal development.

Reply:
yeah most of the code tries to abstract out the ray remote calls, so it shouldn't be hard in theory